### PR TITLE
frzyc jumped the gun now we need to have more stuff here

### PR DESCRIPTION
**Pull request by frzyc**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+#
+
+# source code
+*.php text
+*.css text
+*.sass text
+*.scss text
+*.less text
+*.styl text
+*.js text eol=lf
```
Comment (nguyentvan7) [inline .gitattributes:22]: `.ts` and `.tsx` found dead in a ditch

Comment (frzyc) [reply .gitattributes:22]: LOL i just assumed they had .ts files

Comment (frzyc) [reply .gitattributes:22]: Technically .ts and .tsx should be covered by `* text=auto`, I just switched between tooflesswolf's PR and this one and no funky LF files got generated

**Review by nguyentvan7 — APPROVED** [review]

File: .gitattributes (hunk 1 above)
Comment (nguyentvan7) [reply .gitattributes:22]: Seems like the rest of the text endings are unneeded, but I'm too lazy to figure it out

**Comment by nguyentvan7** [comment]
also, the PR title should be "frzyc jumped the gun". You approved, a merge was inevitable :D

**Pull request merged**